docs/api-markdown-documenter/index.js

```diff
@@ -4,16 +4,33 @@
  */
 
 const chalk = require("chalk");
-
+const yaml = require("js-yaml");
+const fs = require("fs");
 const { renderApiDocumentation } = require("./render-api-documentation");
 
-renderApiDocumentation().then(
-	() => {
-		console.log(chalk.green("API docs written!"));
-		process.exit(0);
-	},
-	(error) => {
-		console.error("API docs could not be written due to an error:", error);
-		process.exit(1);
-	},
-);
+const renderMultiVersion = process.argv[2];
+
+let versions;
+
+try {
+	versions = yaml.load(fs.readFileSync("./data/versions.yaml", "utf8"));
+} catch (e) {
+	console.log(e);
+}
+
+docVersions = renderMultiVersion
+	? versions.params.previousVersions
+	: versions.params.currentVersion;
+
+docVersions.forEach((version) => {
```

Comment:
We should probably use `Promise.all` here, rather than running each promise in sequence.